RecipeMatch/View Controllers/PreferencesViewController.m

```diff
@@ -0,0 +1,133 @@
+    
+    self.cuisineMenu = [[ManaDropDownMenu alloc] initWithFrame:frame title:@"no preference"];
+    self.cuisineMenu.numberOfRows = 19;
+    self.cuisineMenu.textOfRows = @[@"American", @"Asian", @"British",@"Caribbean",@"Central Europe",@"Chinese", @"Eastern Europe", @"French", @"Indian", @"Italian", @"Japanese", @"Kosher", @"Mediterranean", @"Mexican", @"Middle Eastern", @"Nordic", @"South American", @"South East Asian", @"no preference"];
+    self.cuisineMenu.activeColor = UIColorFromRGB(0x80CB99);
+    self.cuisineMenu.heightOfRows = 30;
+    self.cuisineMenu.delegate = self;
+    [self.view addSubview:self.cuisineMenu];
+    
+    
+    self.healthMenu = [[ManaDropDownMenu alloc] initWithFrame:CGRectOffset(frame, 0, 80) title:@"no preference"];
+    self.healthMenu.numberOfRows = 10;
+    self.healthMenu.textOfRows = @[@"vegan", @"vegetarian", @"tree-nut-free",@"low-sugar",@"shellfish-free",@"pescatarian", @"paleo", @"gluten-free", @"fodmap-free", @"no preference"];
+    self.healthMenu.activeColor = UIColorFromRGB(0x80CB99);
+    self.healthMenu.heightOfRows = 30;
+    self.healthMenu.delegate = self;
+    [self.view addSubview:self.healthMenu];
+
+    self.dietMenu = [[ManaDropDownMenu alloc] initWithFrame:CGRectOffset(frame,0,160) title:@"no preference"];
+    self.dietMenu.numberOfRows = 7;
+    self.dietMenu.textOfRows = @[@"balanced", @"high-fiber", @"high-protein",@"low-carb",@"low-fat",@"low-sodium",@"no preference"];
+    self.dietMenu.activeColor = UIColorFromRGB(0x80CB99);
+    self.dietMenu.heightOfRows = 30;
+    self.dietMenu.delegate = self;
+    [self.view addSubview:self.dietMenu];
+    
+    self.mealMenu = [[ManaDropDownMenu alloc] initWithFrame:CGRectOffset(frame, 0, 240) title:@"no preference"];
+    self.mealMenu.numberOfRows = 6;
+    self.mealMenu.textOfRows = @[@"breakfast", @"dinner", @"lunch",@"snack",@"teatime",@"no preference"];
+    self.mealMenu.activeColor = UIColorFromRGB(0x80CB99);
+    self.mealMenu.heightOfRows = 30;
+    self.mealMenu.delegate = self;
+    [self.view addSubview:self.mealMenu];
+}
+
+- (void)dropDownMenu:(CCDropDownMenu *)dropDownMenu didSelectRowAtIndex:(NSInteger)index{
+    if (dropDownMenu == self.cuisineMenu) {
+        self.cuisineLabel = @"&cuisineType=";
+        self.cuisineLabel = [self.cuisineLabel stringByAppendingString:((ManaDropDownMenu *)dropDownMenu).title];
+    }
+    if (dropDownMenu == self.healthMenu) {
+        self.healthLabel = @"&health=";
+        self.healthLabel = [self.healthLabel stringByAppendingString:((ManaDropDownMenu *)dropDownMenu).title];
+    }
+    if (dropDownMenu == self.dietMenu) {
+        self.dietLabel = @"&diet=";
+        self.dietLabel = [self.dietLabel stringByAppendingString:((ManaDropDownMenu *)dropDownMenu).title];
+    }
+    if (dropDownMenu == self.mealMenu) {
+        self.mealLabel = @"&mealType=";
+        self.mealLabel = [self.mealLabel stringByAppendingString:((ManaDropDownMenu *)dropDownMenu).title];
+    }
```

Comment:
Better either be in a if-else style, or with a return at the end of every if statement